python/ct/test/time_utils.py

```diff
@@ -0,0 +1,27 @@
+"""Utilities for testing time-sensitive code."""
+
+import contextlib
+import os
+import time
+
+
+def get_timezone_environ():
+    return os.environ.get("TZ", '')
+
+def set_timezone_environ(new_timezone):
+    os.environ["TZ"] = new_timezone
+    time.tzset()
+
+@contextlib.contextmanager
```

Comment:
Nit: Are you going to re-use this context manager somewhere else?
If not, I how about folding timezone setting into the setUp / tearDown of the only test class that currently uses it?